partials/tabs/projects.html

```diff
@@ -14,11 +14,13 @@
           </a>
         </div>
       </div>
-      <div ng-class="{'display-none':!lc.displayFilters}" ng-click="toggleFiltersDisplay()">
-        <i class="fa fa-times close-filters"></i>
-      </div>
       <div class="all-filters-option apply-flex center-content fade-in" ng-class="{'display-none':!lc.displayFilters}">
-        <h6><b>Filter Projects</b></h6>
+        <span> 
+          <div ng-class="{'display-none':!lc.displayFilters}" ng-click="toggleFiltersDisplay()">
+            <i class="fa fa-times close-filters"></i>
+          </div>
+          <h4><b>Filter Projects</b></h4>
```

Comment:
You have changed h6 to h4 any specific reason for that? Can you please attach a screenshot with the changes

Reply:
@anshalshukla I have already attached the screenshot above. The reason for the change in h tag is that it was not looking good. The font of the header of the dialog box should be greater or equal to the rest of the font in the dialog box. Hope it is fine.